simple-repository-API/Controllers/WeatherForecastController.cs

```diff
@@ -30,7 +30,7 @@ public IEnumerable<WeatherForecast> Get()
             .ToArray();
         }
         [HttpGet("obojetnie")]
-        public List<Student> Get2([FromBody] Student dupa)
+        public List<Student> Get2([FromQuery] int Id_Student)
```

Comment:
[FromQuery nie potrzebne, defaultowo bedzie jako from query wiec nei trzeba pisać

Reply:
Nazwałbym to studentId